resources/views/student-council/anonymous-questions/index.blade.php

```diff
@@ -45,7 +45,7 @@ class="right" :text="__('anonymous_questions.export')" />
                 @endforeach
             </ul>
 
-            @if(!$semester->isClosed())
+            @if($semester->isCurrent() || !$semester->isClosed())
             <div class="row" style="margin: 0">
                 <x-input.button :href="route('anonymous_questions.create', $semester)"
                         class="right green" :text="__('anonymous_questions.create_question')" />
```

Comment:
This is the idea. That way, the council could add questions to the current and future semesters, but collegists would not see anything from that. There, we can simply proceed as you've suggested; the appropriate semester can easily be determined for an open evaluation form.